### PR TITLE
proceed to stage 1

### PR DESCRIPTION
**Pull request by younies**

### CODE REVIEW

**Comment by sffc** [comment]
We should wait for approval from both TG1 (the meeting yesterday) and TG2 (the monthly ECMA-402 meeting).  We have TG1 approval, but not TG2.  So, please don't merge this PR until we have sign-off at next week's TG2 meeting.

**Review by sffc — CHANGES_REQUESTED** [review]
Pending TG2 approval

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ internationalized software.
 In the US and a handful of other countries, temperature is measured in degrees
 Fahrenheit. The rest of the world uses degrees Celsius. Some countries measure
 distances and lengths in miles, yards or feet, and inches, others use
-kilometers, meters, and centimeters. The Scandinavian Mile is a common unit of
+kilometers, meters, and entimeters. The Scandinavian Mile is a common unit of
```
Comment (sffc) [inline README.md]: Why did this change?

Comment (younies) [reply README.md]: > We should wait for approval from both TG1 (the meeting yesterday) and TG2 (the monthly ECMA-402 meeting). We have TG1 approval, but not TG2. So, please don't merge this PR until we have sign-off at next week's TG2 meeting.

We have received the approval a moment ago :)

Comment (younies) [reply README.md]: Done !

File: index.html [hunk 2]
```diff
@@ -1818,7 +1818,7 @@
...
+</style></head><body><div id="menu-toggle">☰</div><div id="menu-spacer"></div><div id="menu"><div id="menu-search"><input type="text" id="menu-search-box" placeholder="Search..."><div id="menu-search-results" class="inactive"></div></div><div id="menu-pins"><div class="menu-pane-header">Pins</div><ul id="menu-pins-list"></ul></div><div class="menu-pane-header">Table of Contents</div><div id="menu-toc"><ol class="toc"><li><span class="item-toggle-none"></span><a href="#sec-demo-clause" title="This is an emu-clause"><span class="secnum">1</span> This is an emu-clause</a></li><li><span class="item-toggle-none"></span><a href="#sec-copyright-and-software-license" title="Copyright &amp; Software License"><span class="secnum">A</span> Copyright &amp; Software License</a></li></ol></div></div><div id="spec-container"><h1 class="version first">Stage -1 Draft / June 5, 2020</h1><h1 class="title">Proposal Title Goes Here</h1>
```
Comment (younies) [inline index.html]: @sffc , shall we change the stage and the date here too?

**Review by sffc — APPROVED** [review]

**Pull request merged**